deps/curl/lib/cf-https-connect.c

```
@@ -240,123 +241,320 @@ static CURLcode baller_connected(struct Curl_cfilter *cf,
+    CURL_TRC_CF(data, cf, "%s baller failed, starting %s",
+                ctx->ballers[0].name, ctx->ballers[1].name);
     return TRUE;
   }
-  elapsed_ms = curlx_ptimediff_ms(&now, &ctx->started);
+
+  elapsed_ms = curlx_ptimediff_ms(Curl_pgrs_now(data), &ctx->started);
   if(elapsed_ms >= ctx->hard_eyeballs_timeout_ms) {
-    CURL_TRC_CF(data, cf, "hard timeout of %" FMT_TIMEDIFF_T "ms reached, "
-                "starting %s",
-                ctx->hard_eyeballs_timeout_ms, ctx->ballers[idx].name);
+    CURL_TRC_CF(data, cf, "%s inconclusive after %" FMT_TIMEDIFF_T ", "
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If CURL_FORMAT_CURL_OFF_T is a macro then please configure it.

(unknownMacro)